src/parser/sparqlParser/SparqlQleverVisitor.h

```diff
@@ -109,19 +93,16 @@ class SparqlQleverVisitor {
   // meaning of blank and anonymous nodes is different.
   bool isInsideConstructTriples_ = false;
 
-  // Member start time is needed for the NOW expression. All calls within
-  // the query execution reference it. The underlying date time format is e.g.:
-  // 2011-01-10T14:45:13.815-05:00
-  std::string startTime_;
-
  public:
   SparqlQleverVisitor() = default;
   explicit SparqlQleverVisitor(
       PrefixMap prefixMap,
       DisableSomeChecksOnlyForTesting disableSomeChecksOnlyForTesting =
           DisableSomeChecksOnlyForTesting::False)
       : prefixMap_{std::move(prefixMap)},
-        disableSomeChecksOnlyForTesting_{disableSomeChecksOnlyForTesting} {}
+        disableSomeChecksOnlyForTesting_{disableSomeChecksOnlyForTesting} {
+    setStartTime();
```

Comment:
The default constructor is not calling `setStartTime()`.

The more correct way would be to
define a static helper function that returns `now` appropriately formatted, and then simply define the member as 
`std::string startTime_ = currentTimeAsXsdString()`.

Then all the constructors do automatically the right thing.